src/region.js

```diff
@@ -223,7 +176,6 @@ var Region = MarionetteObject.extend({
     return Backbone.$(el, this.getValue(this.getOption('parentEl')));
   },
 
-  // Replace the region's DOM element with the view's DOM element.
   _replaceEl: function(view) {
     // empty el so we don't save any non-destroyed views
     this.$el.contents().detach();
```

Comment:
I can't come up with a scenario where there would be an un-destroyed view here.  `empty()` takes care of detaching views when `preventDestroy` is true.  @ahumphreys87 @paulfalgout any ideas why this is necessary?